src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    autocomplete="off"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:
+          <input class="input" type="text"
+          name="Surname" id="surname"
+          required
+            >
+        </label> <br>
```

Comment:
```suggestion
        </label>
        <br />
```
Один тег - одна строка, и вроде br самозакрывающийся, там должна стоять `/` в конце